packages/svelte-ui/components/Button.svelte

```diff
@@ -1,47 +1,41 @@
 <script lang="ts">
+  import { createEventDispatcher } from 'svelte';
   import 'iconify-icon';
-  import { onDestroy } from 'svelte';
-  export let bg: string = 'primary';
-  export let size: string = 'sm';
-  export let icon: string = null;
   export let CTA: string;
-  export let isOutline: boolean = false;
-  export let onClick: Function;
-
-  let isProcessing: boolean = false;
+  export let icon: string = null;
+  export let type:
```

Comment:
`type` might be too generic. Also, it's a good convention to capitalize type names in a manner that's similar to classes/ components.

`type` could be changed to `ButtonType`.